test/helpers/position_assertions.cc

```diff
@@ -10,6 +10,7 @@
 #include "main/lsp/LSPConfiguration.h"
 #include "test/helpers/lsp.h"
 #include "test/helpers/position_assertions.h"
+#include <filesystem>
```

Comment:
We don't use `<filesystem>` elsewhere in Sorbet. Is this ready yet? I don't think this works on all the platforms we want yet.

We have wrappers for filesystem-y things that we want in `common/FileSystem.{h,cc}`

cc @jvilk-stripe would know more here.

Reply:
Yup, +1; put any new needed FS functionality into the file system files.

Reply:
Yep, I only use path manipulation functionality from the library (and no actual system case wrapper) which are lacking currently in Sorbet. I was going to implement them back in that layer once the core approach was validated.